projects/plugins/crm/src/automation/class-automation-workflow.php

```diff
@@ -67,7 +83,7 @@ class Automation_Workflow {
 	 * The workflow active status.
 	 *
 	 * @since $$next-version$$
-	 * @var bool
+	 * @var int
```

Comment:
I think we can keep it as boolean, it won't affect to the DB/Repository, it has the responsibility to do the conversion bool<=>int, casting the value. 

Our logic should remain as boolean, and don't allow values for `$active` distinct to 0/1.